queue/management/commands/queue_ungraded_submissions.py

```diff
@@ -0,0 +1,117 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.db.models import Q
+from django.utils import timezone
+from datetime import timedelta
+import dateutil.parser
+import pprint
+
+from queue.models import Submission
+from queue.producer import push_to_queue
+
+log = logging.getLogger(__name__)
+
+
+def parse_iso_8601_string(iso_string):
+    return dateutil.parser.parse(iso_string)
```

Comment:
Since we are specifying the format of the input string, I think we can get away with using `datetime.strptime()` instead of pulling in an additional dependency.

Reply:
Sounds good. `dateutil` is more permissive about the values it can parse, but I guess that's not really that important